slack-go.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+define("GOSERVER", "http://go.brandwatch.com/");
```

Comment:
Constants should be in screaming snake-case.
https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-1-basic-coding-standard.md#41-constants